klippy/extras/probe.py

```diff
@@ -346,18 +360,21 @@ def start_probe(self, params):
             self.probe_offsets = (0., 0., 0.)
             self._manual_probe_start()
             return
+        else:
+            self.lift_speed = probe.lift_speed
         # Perform automatic probing
         self.lift_speed = probe.get_lift_speed(params)
         self.probe_offsets = probe.get_offsets()
         if self.horizontal_move_z < self.probe_offsets[2]:
             raise self.gcode.error("horizontal_move_z can't be less than"
                                    " probe's z_offset")
+        probe.start_multi_probes()
         while 1:
-            done = self._move_next()
-            if done:
+            if self._move_next():
```

Comment:
This change seems unrelated to the focus of this patch and I'd say it would be better to not mix it in.  (I'm not sure there's a big gain by rewording these lines.

Reply:
Oops, sorry. My brain somehow couldn't resist...

Reply:
Obsoleted by refactoring